schemas/validation_tests/individual-valid-1.json

```diff
@@ -0,0 +1,41 @@
+{
+    "object_id": {
+        "alias" : "NM305004", 
+        "center_name" : "EBI-TEST",
+        "ega_accession": "EGAI00000000001"
+    },
+    "minimal_public_attributes": {
+        "subject_id": "donor-NM305004",
+        "biological_sex": "male",
+        "phenotype": {
+            "experimental_condition_term": "fibroadenoma",
+            "experimental_condition_curie": "EFO:1000254"
+        }
+    },
+    "individual_relationships": [
+        {
+            "r_type": {
+                "r_type_id": "develops_from"
```

Comment:
Yes, there is always a directionality maintained in all these relationships. The syntax is ``source`` - ``relationship_type`` - ``target``. Hence, even if the relationships appears in both sample and individual, as long as the ``source`` is the sample and the individual the ``target``, it will have the same meaning and be read in a similar fashion: ``sample develops_from individual``